sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/aio/_azure_configuration_client_async.py

```diff
@@ -83,17 +83,31 @@ def __init__(
             base_user_agent=USER_AGENT, **kwargs
         )
 
-        pipeline = kwargs.get("pipeline")
         self._sync_token_policy = AsyncSyncTokenPolicy()
-
-        if pipeline is None:
```

Comment:
Looks like we used to allow user to use customized pipeline and we are changing the behavior?

Reply:
We still support using the pipeline from user, the `pipeline` kwarg is passing to the generated client `AzureAppConfiguration`.